### PR TITLE
C cdecclarationupdates

### PR DESCRIPTION
**Pull request by constancecerf**
Comments should be reflected in this document: https://docs.google.com/document/d/1Ow0rxheWLl78CTvJo8mSS_2Qhs8vtdZmnuPGBR8oBDc/edit?usp=sharing

Service questions have been updated including validation messages and any mention of 'accredited' services in the flow.

### CODE REVIEW

**Comment by CrystalPea** [comment]
@constancecerf some questions were deleted in this pull request and we have to let the tests know. We can try to do it together if you want :).

**Comment by Wynndow** [comment]
This will need a version bump, it's probably worth making the changes in this PR for the questions that need the logic changes too.

File: frameworks/g-cloud-10/questions/services/browsersSupported.yml [hunk 1]
```diff
@@ -15,8 +15,10 @@ options:
     value: ie8
   - label: Internet Explorer 9
     value: ie9
-  - label: Internet Explorer 10+
-    value: ie10+
+  - label: Internet Explorer 10
+    value: ie10
+  - label: Internet Explorer 11
```
Comment (Wynndow) [inline frameworks/g-cloud-10/questions/services/browsersSupported.yml]: Should this be `Internet Explorer 11+`

File: frameworks/g-cloud-9/manifests/declaration.yml [hunk 2]
```diff
@@ -14,7 +14,6 @@
   questions:
     - servicesHaveOrSupport
     - servicesDoNotInclude
-    - payForWhatUse
```
Comment (Wynndow) [inline frameworks/g-cloud-9/manifests/declaration.yml]: We probably shouldn't change g-9 stuff?

Comment (CrystalPea) [reply frameworks/g-cloud-9/manifests/declaration.yml]: good spot, changing that back now 👍

File: frameworks/g-cloud-10/questions/services/browsersSupported.yml (hunk 1 above)
Comment (CrystalPea) [reply frameworks/g-cloud-10/questions/services/browsersSupported.yml]: I think you are correct, I will change that :)

Comment (Wynndow) [reply frameworks/g-cloud-10/questions/services/browsersSupported.yml]: Yep, that's true. As you were.

**Comment by samuelhwilliams** [comment]
Have updated to keep the keys in line with G9 services. Are you happy that all your content changes are in this @constancecerf?

**Pull request merged**